rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -234,12 +234,15 @@ def get_bucket_notification(rgw_s3_client, bucketname, empty=False):
     )
 
 
-def verify_event_record(event_type, bucket, event_record_path, ceph_version):
+def verify_event_record(event_type, bucket, event_record_path, ceph_version, config):
```

Comment:
@viduship , verify_event_record is called in some other places also
so, could you make config=None here

Reply:
Done